src/backend/config.ts

```diff
@@ -323,7 +324,9 @@ class GlobalConfigV0 extends GlobalConfig {
       winePrefix: isWindows ? '' : defaultWinePrefix,
       wineVersion: defaultWine,
       enableEsync: true,
-      enableFsync: true
+      enableFsync: isLinux,
```

Comment:
fsync uses a linux specific API and can only should be enabled by default on linux

Reply:
This may be nitpicky, but I don't think this should be in this PR